backend/app/routes/appRoutes.js

```diff
@@ -1,10 +1,10 @@
-const express = require("express");
-const { getRestaurantById, getRestaurantByName, hostnameCheck } = require("../controllers/appController");
+const express = require('express');
+const { getRestaurantByIdController, getRestaurantByNameController, getRestaurantByHostnameController } = require('../controllers/appController');
 
 const router = express.Router();
 
-router.get("/restaurantName/:restaurantName", getRestaurantByName);
-router.get("/restaurantId/:restaurantId", getRestaurantById);
-router.post("/hostnameCheck", hostnameCheck);
+router.get('/restaurantId/:restaurantId', getRestaurantByIdController);
+router.get('/restaurantName/:restaurantName', getRestaurantByNameController);
+router.post('/hostnameCheck', getRestaurantByHostnameController);
 
 module.exports = router;
```

Comment:
The code patch seems to be a refactoring of the existing codebase. It changes the naming convention of functions and endpoints, which is a good practice for code readability and maintainability. The following improvements can be suggested:

1. The imports can further be simplified by destructuring the "express" module.

2. The endpoint names seem ambiguous, it's better to use descriptive names for endpoints that clearly describe the functionality.

3. It can add more comments to describe the purpose of each function or endpoint.

Other than these improvements, the code patch seems to be correct and not introducing any bug risks.